src/clusterfuzz/_internal/bot/tasks/utasks/symbolize_task.py

```diff
@@ -298,5 +295,50 @@ def get_symbolized_stacktraces(testcase_file_path, testcase,
   return symbolized, stacktrace
 
 
+HANDLED_ERRORS = [
+    uworker_msg_pb2.ErrorType.SYMBOLIZE_BUILD_SETUP_ERROR,  # pylint: disable=no-member
+    uworker_msg_pb2.ErrorType.UNHANDLED  # pylint: disable=no-member
+] + setup.HANDLED_ERRORS
+
+
 def utask_postprocess(output):
-  del output
+  """Handle the output from utask_main."""
+  if output.error_type is not None:
+    uworker_handle_errors.handle(output, HANDLED_ERRORS)
+    return
+
+  symbolize_task_output = output.symbolize_task_output
+
+  # Update crash parameters.
+  testcase = data_handler.get_testcase_by_id(output.uworker_input.testcase_id)
+  testcase.crash_type = symbolize_task_output.crash_type
+  testcase.crash_address = symbolize_task_output.crash_address
+  testcase.crash_state = symbolize_task_output.crash_state
+  testcase.crash_stacktrace = symbolize_task_output.crash_stacktrace
+
+  if not symbolize_task_output.symbolized:
+    data_handler.update_testcase_comment(
+        testcase, data_types.TaskState.ERROR,
+        'Unable to reproduce crash, skipping '
+        'stacktrace update')
+    # TODO(alhijazi): I am conserving previous behaviour, but shouldn't we
```

Comment:
I believe this is to prevent retries of symbolization in these cases. This flag is used to control whether or not we want to attempt creating a symbolize task at all (e.g. https://github.com/google/clusterfuzz/blob/41a011f9cba88bdbefcae7bf258aea55cf05fc6f/src/clusterfuzz/_internal/bot/tasks/task_creation.py#L105). This is also the only place this flag is used.

Reply:
Noted, thanks! 
Will remove the TODO.